semcore/wizard/src/Wizard.jsx

```diff
@@ -43,42 +74,48 @@ class WizardRoot extends Component {
       active: props.step === this.asProps.step,
       number,
       getI18nText: this.asProps.getI18nText,
+      uid: this.asProps.uid,
     };
   }
 
   componentDidUpdate(prevProps) {
     if (prevProps.step === this.asProps.step) return;
     setTimeout(() => {
       if (prevProps.step === this.asProps.step) return;
-      if (document.activeElement !== document.body) return;
-      if (!this.asProps.visible) return;
-      if (!this.modalRef.current) return;
-      setFocus(this.modalRef.current, document.body);
+      setFocus(this.contentRef.current);
     }, 1);
   }
 
   render() {
     const SWizard = this.Root;
-    const { Children, styles } = this.asProps;
+    const { Children, styles, uid, step } = this.asProps;
 
     this._steps.clear();
 
     return sstyled(styles)(
-      <SWizard render={Modal} ref={this.modalRef}>
+      <SWizard
+        render={Modal}
+        aria-label={undefined}
+        ref={this.modalRef}
+        aria-labelledby={`${uid}-step-${step}`}
+      >
         <Children />
       </SWizard>,
     );
   }
 }
 
 function Sidebar(props) {
-  const { Children, styles, title } = props;
+  const { Children, styles, title, uid } = props;
   const SSidebar = Root;
   const SSidebarHeader = 'h2';
+  const SSidebarMenu = 'div';
   return sstyled(styles)(
-    <SSidebar render={Box} role='menu'>
-      {title && <SSidebarHeader>{title}</SSidebarHeader>}
-      <Children />
+    <SSidebar render={Box}>
```

Comment:
Sidedar has an HTML title that equals to the text of the Sidebar heading. Because of that there's always a system tooltip when my mouse is over the Sidebar. And the Screen reader announces the Sidebar using this title, which is incorrect, because this title describes the whole modal, not the Sidebar. 

Can we suppress passing the "title" attribute to the resulting div element?

<img width="479" alt="image" src="https://github.com/user-attachments/assets/0595ab13-62c1-4846-8d7a-764fd2c40b23"> <img width="287" alt="image" src="https://github.com/user-attachments/assets/40b541d0-c0af-4cff-a2a2-7d8a955fe364">